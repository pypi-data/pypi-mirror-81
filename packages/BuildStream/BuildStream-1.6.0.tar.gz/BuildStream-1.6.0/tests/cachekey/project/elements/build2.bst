# This tests that sandbox build-uid / build-gid
# contributions to the cache key do not regress.
#
kind: manual
sources:
- kind: local
  path: files/local

config:
  build-commands:
  - make

sandbox:
  build-uid: 20
  build-gid: 20
