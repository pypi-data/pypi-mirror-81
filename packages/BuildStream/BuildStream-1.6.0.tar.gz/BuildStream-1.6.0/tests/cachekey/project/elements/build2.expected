7104190005529a3fe8a1ae6beb794b6a371312b915b35bec360af40a52bd3778